build/gulpfile.js

```diff
@@ -56,12 +58,17 @@ module.exports = function(gulp, config, pluginName){
         module: 'commonjs',
         declarationFiles: true,
         noExternalResolve: false,
-        removeComments: true
-      })))
-      .on('error', plugins.notify.onError({
```

Comment:
Thanks, I have learned to really like the notifications as I have the console running in some back window and have my IDE in fullscreen, so I just listen for a ding and the toasts fly across my screen. I don't have to look at console window ;)